docs/relnotes.md

```diff
@@ -6,6 +6,8 @@ hide:
 # Release Notes
 
 This is the list of changes to Taipy releases as they were published.
+Please refer to [upgrades](./legacy-version-migration.md) page for
+details on how to upgrade Taipy versions.
```

Comment:
```suggestion
details on how to upgrade Taipy version.
```
No `s` right?

Reply:
"Upgrading" a Python module is not something we want to document. That would be pip.
"Migrating an application", on the other hand, certainly is one of our concerns.

I would phrase that: "how to migrate your application from 2.0 to 2.1"
and move that reference to the Release Notes for 2.1 only.

Reply:
Done now